README.md

```diff
@@ -12,6 +12,32 @@ Gramps Web API is the backend of [Gramps Web](https://www.grampsweb.org/), a gen
 - Developer documentation for Gramps Web API: https://www.grampsweb.org/dev-backend/
 - Documentation for Gramps Web: https://www.grampsweb.org
 
+## OIDC Configuration
```

Comment:
This is not the right place to document configuration. Please submit a PR to https://github.com/gramps-project/gramps-web-docs.